js-exercises/is-triangle/isTriangle.js

```diff
@@ -0,0 +1,20 @@
+
+
+function isTriangle(...sides) {
+  if (sides.length > 3) {
+    throw new Error(
+      `Number of paramter passed is more than 3, Triangle has only three sides`
```

Comment:
No spelling mistakes please. `paramter` -> `parameters`.

Reply:
will make the changes, i accept the review, yes, i should check the error messages properly too.

Reply:
I have made the changes and made a git push to the same branch, is it correct what i am doing ! please correct me if i am wrong.